test-d/fastly:backend.test-d.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference path="../types/fastly:backend.d.ts" />
+import { Backend, BackendConfiguration } from "fastly:backend";
+import {expectError, expectType} from 'tsd';
+
+// Backend
+{
+    expectError(Backend())
+    expectError(new Backend())
+    expectError(new Backend({name: 'eu'}))
+    expectError(new Backend({target: 'www.example.com'}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com'}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', hostOverride: 'example.com', }))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', connectTimeout: 5000, }))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', firstByteTimeout: 5000, }))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', betweenBytesTimeout: 5000, }))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', useSSL: true,}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', tlsMinVersion: 1.2,}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', tlsMaxVersion: 1.2,}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', certificateHostname: 'example.com',}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', checkCertificate: false,}))
+    expectType<Backend>(new Backend({name: 'eu', target: 'www.example.com', caCertificate: '',}))
```

Comment:
Is the empty string a meaningful value for `caCertificate`, or is it the same as providing `undefined`?

Reply:
> Is the empty string a meaningful value for `caCertificate`, or is it the same as providing `undefined`?

Providing an empty string will throw a TypeError at runtime with the message `"Backend constructor: caCertificate can not be an empty string"`